jobs/awslogs/spec

```diff
@@ -17,6 +17,8 @@ packages:
 - awslogs
 - awslogs-python3
 - inotify-tools
+- openssl
+- zlib
```

Comment:
Do we actually need this here? or is it because it's a dep of a dep that we can leave it out? Do we actually need these libraries outside of the packing / complication step? @sharms